android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/SearchState.kt

```diff
@@ -51,6 +52,18 @@ data class SearchState(
 val SearchState.searchEngines: List<SearchEngine>
     get() = (regionSearchEngines + additionalSearchEngines + customSearchEngines + applicationSearchEngines)
 
+/**
+ * The list of search engine shortcuts to be available for quick search menu.
+ * // TODO maybe move it into ext file
+ */
+val SearchState.searchEngineShortcuts: List<SearchEngine>
```

Comment:
For similar reasons above, let's move this extension function to the fenix layer.

Meta: From a discussion with Mike offline, we also realized that sorting of this list is also an application-level customization for displaying in the app.

Reply:
implemented